code/game/machinery/recharger.dm

```diff
@@ -10,6 +10,11 @@
 	var/obj/item/charging = null
 	var/static/list/allowed_devices = typecacheof(list(/obj/item/weapon/gun/energy, /obj/item/weapon/melee/baton, /obj/item/ammo_box/magazine/recharge, /obj/item/device/modular_computer))
 	var/recharge_coeff = 1
+	var/icon_state_off = "rechargeroff"
+	var/icon_state_charged = "recharger2"
+	var/icon_state_charging = "recharger1"
+	var/icon_state_idle = "recharger0"

```

Comment:
These aren't used anywhere and you shouldn't add new vars for this sort of thing anyway